FreshRSS/FreshRSS
# Fix #3835: theme switcher broken when theme is not available

Closes #3835

Changes proposed in this pull request:
- an error message is display when the theme is not available
- user can use the prev-button to choose an existing theme
- when user does not chose another theme and clicks on safe, than it safes "Origine" theme.

![grafik](https://user-images.githubusercontent.com/1645099/136713104-e82f3916-9769-42a5-a603-e908aea8edd7.png)


How to test the feature manually:
1. delete/rename the chosen theme folder in `p/themes`
2. go to settings -> display
3. see the red error message

Pull request checklist:
- [x] i18n: English
- [x] i18n German
- [ ] i18n French
- other i18n: please give me a translation
